.gitignore

```diff
@@ -0,0 +1,6 @@
+# Django stuff:
+*.log
+local_settings.py
+db.sqlite3
+db.sqlite3-journal
+*.pyc
```

Comment:
Once you update .pyc file in .gitignore you have to remove it from git cache.  see https://stackoverflow.com/a/19095988

Reply:
I have removed all the pyc files, database file and migrations in this pull.